src/mongo/registerMongoCommands.ts

```diff
@@ -18,7 +18,7 @@ import { MongoCollectionTreeItem } from "./tree/MongoCollectionTreeItem";
 import { MongoDatabaseTreeItem } from "./tree/MongoDatabaseTreeItem";
 import { MongoDocumentTreeItem } from "./tree/MongoDocumentTreeItem";
 
-const connectedDBKey: string = 'ms-azuretools.vscode-cosmosdb.connectedDB';
+const connectedDBKey: string = 'ms-azuretools.vscode-azuredatabases.connectedDB';
```

Comment:
This means users will lose the cache of their connected database. Since the key is completely internal and not displayed to the user - I would keep it as-is. Plus the feature is cosmos specific anyways